AssetBundleManager/AssetBundleDownloader.cs

```diff
@@ -78,13 +78,13 @@ private IEnumerator Download(AssetBundleDownloadCommand cmd, int retryCount)
             UnityWebRequest req;
             if (cachingDisabled || (cmd.Version <= 0 && cmd.Hash == DEFAULT_HASH)) {
                 Debug.Log(string.Format("GetAssetBundle [{0}].", uri));
-                req = UnityWebRequest.GetAssetBundle(uri);
+                req = UnityWebRequestAssetBundle.GetAssetBundle(uri);
```

Comment:
UnityWebRequestAssetBundle is new to Unity 2018.  In order to support Unity 2017 (which is what Sad Panda is using) or older this will need to be surrounded with `#if UNITY_2018_1_OR_NEWER`